### PR TITLE
Added Navbar layout

### PR DESCRIPTION
**Pull request by garr741**
@bwencke @mhoc @ashleyzeg

### CODE REVIEW

File: imports/ui/components/App.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ class App extends Component {
   render() {
     return (
       <div>
-        <Navbar />
+        <Navbar/>
```
Comment (bwencke) [inline imports/ui/components/App.jsx]: I believe it is customary to put a space

File: imports/ui/components/Navbar.jsx [hunk 2]
```diff
@@ -1,11 +1,106 @@
...
+          fullWidth={true}
+        />
+      </div>
+    )
+  }
+}
+
+const toolbarGroupStyle = {
+  margin: 'auto',
+  width: '90%'
```
Comment (bwencke) [inline imports/ui/components/Navbar.jsx]: no trailing comma?

File: imports/ui/components/Navbar.jsx [hunk 3]
```diff
@@ -1,11 +1,106 @@
...
+  paddingRight: '5px',
+  backgroundColor: '#EEEEEE',
+  width: '100%',
+  display: 'inline-block'
+
+}
+
+const iconStyles = {
+  width: '90px'
+};
```
Comment (bwencke) [inline imports/ui/components/Navbar.jsx]: semicolon?

File: imports/ui/components/Navbar.jsx [hunk 4]
```diff
@@ -1,11 +1,106 @@
...
+  paddingLeft: '5px',
+  paddingRight: '5px',
+  backgroundColor: '#EEEEEE',
+  width: '100%',
+  display: 'inline-block'
+
+}
+
+const iconStyles = {
+  width: '90px'
```
Comment (bwencke) [inline imports/ui/components/Navbar.jsx:105]: comma

File: imports/ui/components/Navbar.jsx [hunk 5]
```diff
@@ -1,11 +1,106 @@
...
+  margin: 'auto',
+  width: '90%'
+}
+
+const searchAreaStyle = {
+  paddingLeft: '5px',
+  paddingRight: '5px',
+  backgroundColor: '#EEEEEE',
+  width: '100%',
+  display: 'inline-block'
```
Comment (bwencke) [inline imports/ui/components/Navbar.jsx]: comma

File: imports/ui/components/App.jsx (hunk 1 above)
Comment (garr741) [reply imports/ui/components/App.jsx]: Yeah, idk what happened there.

File: imports/ui/components/Navbar.jsx (hunk 2 above)
Comment (garr741) [reply imports/ui/components/Navbar.jsx]: i've never done trailing commas before, but i guess i can start.

**Review by bwencke — APPROVED** [review]

**Pull request merged**